pages/contributions/contributions.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Contributing to the JSON Schema documentation
+section: docs
+---
+
+## Welcome
+
+Welcome to the JSON Schema Contributing Guide, and thank you for your interest. Before working on any contributions, we kindly ask you to [read our Code of Conduct](../contributions/code-of-conduct). 
+
+If you would like to contribute to the project, check out the following list of contributions that we accept:
+
+- Documentation: We value documentation and appreciate efforts to help us improve our documents. If you would like to propose modifications we invite you to first [check our documentation board](https://github.com/orgs/json-schema-org/projects/16) and make sure no issue has been created for the changes you're proposing. To create an issue for documentation, please select and fill out the **Documentation** template. 
+- 
+- 
+- If no issue exists, you can [create a GitHub issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-a-repository) and select the *Documentation* template. After that, you can decide whether you want to work on the issue yourself or leave for someone else to work on it. 
```

Comment:
"A- Documentation: We value documentation and appreciate your efforts to help us improve our documents. If you would like to propose modifications, we encourage you to check our documentation board and ensure no issues are created for the proposed changes. To create an issue for documentation, select and fill out the Documentation template."